src/elements/content-sidebar/BoxAISidebarContent.tsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * @file Box AI sidebar component
+ * @author Box
+ */
+import * as React from 'react';
+import flow from 'lodash/flow';
+import { useIntl } from 'react-intl';
+import { AgentsProvider, BoxAiAgentSelectorWithApi } from '@box/box-ai-agent-selector';
+import { IconButton, Text } from '@box/blueprint-web';
+import { Trash } from '@box/blueprint-web-assets/icons/Line';
+// @ts-expect-error - TS2305 - Module '"@box/box-ai-content-answers"' has no exported member 'ApiWrapperProps'.
+import { BoxAiContentAnswers, withApiWrapper, type ApiWrapperProps } from '@box/box-ai-content-answers'
+import SidebarContent from './SidebarContent';
+import { withAPIContext } from '../common/api-context';
+import { withErrorBoundary } from '../common/error-boundary';
+import { withLogger } from '../common/logger';
+import { ORIGIN_BOXAI_SIDEBAR, SIDEBAR_VIEW_BOXAI } from '../../constants';
+import { EVENT_JS_READY } from '../common/logger/constants';
+import { mark } from '../../utils/performance';
+import { BoxAISidebarContext } from './BoxAISidebar';
+
+import messages from '../common/messages';
+import sidebarMessages from './messages';
+
+import './BoxAISidebar.scss';
+
+
+const MARK_NAME_JS_READY: string = `${ORIGIN_BOXAI_SIDEBAR}_${EVENT_JS_READY}`;
+
+mark(MARK_NAME_JS_READY);
+    
+function BoxAISidebarContent(props: ApiWrapperProps) {
+    const { 
+        createSession, 
+        encodedSession, 
+        onClearAction, 
+        getAIStudioAgents,
+        hostAppName,
+        isAIStudioAgentSelectorEnabled, 
+        onSelectAgent, 
+        questions, 
+        sendQuestion, 
+        stopQuestion, 
+        ...rest 
+    } = props;
+    const { formatMessage } = useIntl();
+    const { cache, contentName, elementId, setCacheValue, userInfo } = React.useContext(BoxAISidebarContext);
+    const { questions: cacheQuestions } = cache;
+
+    if (!cache[encodedSession] && encodedSession) {
+        setCacheValue('encodedSession', encodedSession);
+    }
+
+    if (!cache[questions] && questions) {
```

Comment:
Similarly here: shouldn't we check the string `'questions'` key instead of value?